doc/SETUP.md

```diff
@@ -2,8 +2,21 @@
 
 ## 🚀 Quick Start
 
+- Install dependencies\
+  `$ yarn install`
+
 ### 💻 Load extension into browser
 
+- Start development build, which will automatically watch for file changes:
+
+  - Chrome\
+    `$ yarn run dev:chrome`
+  - Firefox\
+    `$ yarn run dev:firefox`
+  - Opera\
+     `$ yarn run dev:opera`
+    **NOTE:** by default, the extension built this way will talk to the development regtest API. In case you want to do manual tests against the production API, add the following `WALLET_CREATE_URL` environment variable to your command: `$ WALLET_CREATE_URL="https://getalby.com/api/users" yarn run dev:your-browser-of-choice`
```

Comment:
```suggestion
    **NOTE:** by default, the extension built this way will talk to the testnet API (which runs under [app.regtest.getalby.com](https://app.regtest.getalby.com/user)). In case you want to do manual tests against the mainnet API, add the following `WALLET_CREATE_URL` environment variable to your command: `$ WALLET_CREATE_URL="https://getalby.com/api/users" yarn run dev:your-browser-of-choice`
```